pennylane_qiskit/converter.py

```diff
@@ -541,7 +543,11 @@ def _function(*args, params: dict = None, wires: list = None, **kwargs):
                 # Iteratively recurse over to build different branches for the condition
                 with qml.QueuingManager.stop_recording():
                     branch_funcs = [
-                        partial(load(branch_inst, measurements=None), params=params, wires=wires)
+                        partial(
```

Comment:
@soranjh and I were trying to figure this out actually... but basically, here's what I did. I put a linter in my local file and ran a `pre-commit` hook the same way you do for the pennylane repo and it reformatted these parts. If I manually change it back to the original formatting and run `black -l 100 tests` on `generalize_ops` it will reformat it back to this. But when I run black on `new_device_feature_branch` nothing gets changed so... :shrug:

My solution is to just ignore it for now, it does make the PR hard to read though, sorry.

Reply:
It's a line I wrote to handle controlled flow branches, which themselves include circuits. So `black -l 100 tests` shouldn't have reformated it. It's fine like this, just wanted to figure out if you were modifying something here apart from formatting.

Reply:
Yeah it shouldn't have reformatted it which is why Soran and I were confused. Not sure what exactly changed 😨